arborx +rocm build fails w/ rocm 4.5.2: cmake error: Could not find a package configuration file provided by "rocthrust"

### Steps to reproduce the issue

`arborx@1.1 +rocm` build fails at CMake stage using:
* spack@develop (d4ca803fc58e127469fd0e0773b89fdc2087fac0 from `Tue Feb 8 21:34:38 2022 +0100`)
* Ubuntu 20.04, X86_64
* GCC 9.3.0
* ROCm 4.5.2

Concrete spec: [arborx-rocm.spec.json.txt](https://github.com/spack/spack/files/8042866/arborx-rocm.spec.json.txt)

Concretization:
```
arborx@1.1%gcc@9.3.0~cuda~ipo+mpi~openmp+rocm+serial~sycl~trilinos build_type=RelWithDebInfo arch=linux-ubuntu20.04-x86_64
    ^cmake@3.22.2%gcc@9.3.0~doc+ncurses+openssl+ownlibs~qt build_type=Release arch=linux-ubuntu20.04-x86_64
        ^ncurses@6.2%gcc@9.3.0~symlinks+termlib abi=none arch=linux-ubuntu20.04-x86_64
            ^pkgconf@1.8.0%gcc@9.3.0 arch=linux-ubuntu20.04-x86_64
        ^openssl@1.1.1m%gcc@9.3.0~docs certs=system arch=linux-ubuntu20.04-x86_64
            ^perl@5.34.0%gcc@9.3.0+cpanm+shared+threads arch=linux-ubuntu20.04-x86_64
                ^berkeley-db@18.1.40%gcc@9.3.0+cxx~docs+stl patches=b231fcc4d5cff05e5c3a4814f6a5af0e9a966428dc2176540d2c05aff41de522 arch=linux-ubuntu20.04-x86_64
                ^bzip2@1.0.8%gcc@9.3.0~debug~pic+shared arch=linux-ubuntu20.04-x86_64
                    ^diffutils@3.8%gcc@9.3.0 arch=linux-ubuntu20.04-x86_64
                        ^libiconv@1.16%gcc@9.3.0 libs=shared,static arch=linux-ubuntu20.04-x86_64
                ^gdbm@1.19%gcc@9.3.0 arch=linux-ubuntu20.04-x86_64
                    ^readline@8.1%gcc@9.3.0 arch=linux-ubuntu20.04-x86_64
                ^zlib@1.2.11%gcc@9.3.0+optimize+pic+shared arch=linux-ubuntu20.04-x86_64
    ^kokkos@3.5.00%gcc@9.3.0~aggressive_vectorization~compiler_warnings~cuda~cuda_constexpr~cuda_lambda~cuda_ldg_intrinsic~cuda_relocatable_device_code~cuda_uvm~debug~debug_bounds_check~debug_dualview_modify_check~deprecated_code~examples~explicit_instantiation~hpx~hpx_async_dispatch~hwloc~ipo~memkind~numactl~openmp~pic+profiling~profiling_load_print~pthread~qthread+rocm+serial+shared~sycl~tests~tuning~wrapper amdgpu_target=gfx908 build_type=RelWithDebInfo std=14 arch=linux-ubuntu20.04-x86_64
        ^hip@4.5.2%gcc@9.3.0~ipo build_type=Release patches=24ca25a169f4bac849464cf7c6dd0885d6475e701fbc4e08e976ba16e4f2e9a6,2a4190477b7d9206b9cd8d70770ba0bc007273cbe54772efb12f9ca2e37c0392 arch=linux-ubuntu20.04-x86_64
        ^hsa-rocr-dev@4.5.2%gcc@9.3.0+image~ipo+shared build_type=Release patches=71e6851d9be8113bfb8d71b66a3171e0d0401bae5e6f161c9e7fe32558261a46 arch=linux-ubuntu20.04-x86_64
        ^llvm-amdgpu@4.5.2%gcc@9.3.0~ipo~link_llvm_dylib~llvm_dylib+openmp+rocm-device-libs build_type=Release patches=d999f3b235e655ee07f6dd2590302082feaa06d32c5c6b53aae9c5cf1e45b644 arch=linux-ubuntu20.04-x86_64
    ^mpich@3.4.2%gcc@9.3.0~argobots+fortran+hwloc+hydra+libxml2+pci+romio~slurm~two_level_namespace~verbs~wrapperrpath device=ch4 netmod=ofi pmi=pmi arch=linux-ubuntu20.04-x86_64
        ^findutils@4.8.0%gcc@9.3.0 patches=440b9543365b4692a2e6e0b5674809659846658d34d1dfc542c4397c8d668b92 arch=linux-ubuntu20.04-x86_64
        ^hwloc@2.7.0%gcc@9.3.0~cairo~cuda~gl~libudev+libxml2~netloc~nvml~opencl+pci~rocm+shared arch=linux-ubuntu20.04-x86_64
            ^libpciaccess@0.16%gcc@9.3.0 arch=linux-ubuntu20.04-x86_64
                ^libtool@2.4.6%gcc@9.3.0 arch=linux-ubuntu20.04-x86_64
                    ^m4@1.4.19%gcc@9.3.0+sigsegv patches=9dc5fbd0d5cb1037ab1e6d0ecc74a30df218d0a94bdd5a02759a97f62daca573,bfdffa7c2eb01021d5849b36972c069693654ad826c1a20b53534009a4ec7a89 arch=linux-ubuntu20.04-x86_64
                        ^libsigsegv@2.13%gcc@9.3.0 arch=linux-ubuntu20.04-x86_64
                ^util-macros@1.19.3%gcc@9.3.0 arch=linux-ubuntu20.04-x86_64
            ^libxml2@2.9.12%gcc@9.3.0~python arch=linux-ubuntu20.04-x86_64
                ^xz@5.2.5%gcc@9.3.0+pic libs=shared,static arch=linux-ubuntu20.04-x86_64
        ^libfabric@1.14.0%gcc@9.3.0~debug~kdreg fabrics=rxm,sockets,tcp,udp arch=linux-ubuntu20.04-x86_64
```

Quickly reproducible w/ cached dependencies using Docker container image:
* `ecpe4s/ubuntu20.04-runner-x86_64:2022-02-01`

```
$> docker run -it ecpe4s/ubuntu20.04-runner-x86_64:2022-02-01

root@a8e01c0ed0cb:/# git clone https://github.com/spack/spack
root@a8e01c0ed0cb:/# (cd spack && git checkout d4ca803fc58e)
root@a8e01c0ed0cb:/# . /spack/share/spack/setup-env.sh

root@a8e01c0ed0cb:/# spack mirror add E4S https://cache.e4s.io
root@a8e01c0ed0cb:/# spack buildcache keys -it
... OK

root@a8e01c0ed0cb:/# spack install --cache-only --include-build-deps --only dependencies -f ./arborx.spec.json
... OK

root@a8e01c0ed0cb:/# spack install --no-cache -f ./arborx.spec.json
...
==> Installing arborx-1.1-rvczpxsyt4gjmti7snqo7dp5jld3wb6v
==> Using cached archive: /spack/var/spack/cache/_source-cache/archive/2b/2b5f2d2d5cec57c52f470c2bf4f42621b40271f870b4f80cb57e52df1acd90ce.tar.gz
==> No patches needed for arborx
==> arborx: Executing phase: 'cmake'
==> Error: ProcessError: Command exited with status 1:
    'cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/arborx-1.1-rvczpxsyt4gjmti7snqo7dp5jld3wb6v' '-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=OFF' '-DCMAKE_INSTALL_RPATH:STRING=/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/arborx-1.1-rvczpxsyt4gjmti7snqo7dp5jld3wb6v/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/arborx-1.1-rvczpxsyt4gjmti7snqo7dp5jld3wb6v/lib64;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/kokkos-3.5.00-zy73auraurlqnxavzfwyqwiovysfgbdy/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/hip-4.5.2-6ypyx5bqnbx5sxlfl7rczrjxgxeyeez7/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/comgr-4.5.2-vform5q7kdm7qlbymgpjo2rwpysytbrx/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/llvm-amdgpu-4.5.2-qb4wlxghtnvdhop7xeyt5657gkbe4gpj/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/hwloc-2.7.0-hswya4sr4io27hwpjsb7oyhgqb6vzy34/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libpciaccess-0.16-4jbn7mznw664dc4mlseouzovsfx6sor5/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxml2-2.9.12-xlbi5rk675mptoyjnuff4q3e2lajs2wr/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libiconv-1.16-zth4ohvj7y3p532iqjw7fptg3kkn3ixv/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/xz-5.2.5-mq4ikt5q4zttamuxqkqe66jlftjbvkf4/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/zlib-1.2.11-ufqpqnpj5gjkfx5sd2o3twwveuowcz2b/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/ncurses-6.2-nxjjphalwbf2ywlxpn74nsfcsrq2bbtd/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libelf-0.8.13-o53xuefowaaefem7z2zfar2ftwgyyezy/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/z3-4.8.14-bvcp2nq4v7f22lbflvor2m6s7wrsymjp/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/hsa-rocr-dev-4.5.2-pmkgo6ew56rh2sm7bwaomvsmji6pwhti/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/hsakmt-roct-4.5.2-bnbzssw7brnkeuvwiuapiohawrcaiaqg/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libdrm-2.4.100-oubvjpazucmha66spq7cpu277ecpl5mh/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libpthread-stubs-0.4-wngdhpdeuvwyfid6b7rkvbhbm22e4dzr/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/numactl-2.0.14-e3evcwcbrfcfnv4j2h55y6n6dciu37wi/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/mesa18-18.3.6-ouol5nccvy6c4piwurted3a5ngxvnr2y/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/expat-2.4.4-pymrfo6q4hnzthoym5uhxwcvriucuoxh/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libbsd-0.11.3-qkquppz2atadbgdtih5wh3cqo7sdrq6w/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libmd-1.0.3-ahm64asdtstsccavq2szva4guzix3uet/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/glproto-1.4.17-sphouu7fembhysc55gv7oovholhmuqcs/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libx11-1.7.0-n7usaickbiq3uwocfg6dt35bpypoldeo/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/inputproto-2.3.2-xbobkehjb4sfrer4oveumgxzln552jgd/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/kbproto-1.0.7-xyjeeo2b6j5m2c3xfi5qvv7alxmci3qh/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxcb-1.14-vlmcy2fthy7dzdwwx6n6njw6pmzwbhfg/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxau-1.0.8-2f3vsx23iochykfdq57nbhzaq6zspipj/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/xproto-7.0.31-kyzpwipokb7xsz5f4x4io5tim4p763fp/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxdmcp-1.1.2-2crqheqqbkcmg4z63zjclasekic5wdls/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/xcb-proto-1.14.1-cimfugd3wmc7sw2wkk2v2fghabo4apg3/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/xextproto-7.3.0-2oagha2n6vf6wmh5lz4o4wxw6zuba6hc/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxext-1.3.3-pv7xlezmya3kzuevm26gxyqo3dlemzxe/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/mpich-3.4.2-b2hsqqqnsiodzp3kw63u425osorilfty/lib;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libfabric-1.14.0-ohufqpc3kq5sd32tcn2pp3t636knmxxd/lib' '-DCMAKE_PREFIX_PATH:STRING=/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/mpich-3.4.2-b2hsqqqnsiodzp3kw63u425osorilfty;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libfabric-1.14.0-ohufqpc3kq5sd32tcn2pp3t636knmxxd;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/kokkos-3.5.00-zy73auraurlqnxavzfwyqwiovysfgbdy;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/hip-4.5.2-6ypyx5bqnbx5sxlfl7rczrjxgxeyeez7;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/roctracer-dev-api-4.5.2-5xq46swsc4x7x4q5wjtpxkhmgi5nkuk2;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/rocminfo-4.5.2-36qqei4wl34ween45yqkaj732lvrywxy;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/mesa18-18.3.6-ouol5nccvy6c4piwurted3a5ngxvnr2y;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxext-1.3.3-pv7xlezmya3kzuevm26gxyqo3dlemzxe;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libx11-1.7.0-n7usaickbiq3uwocfg6dt35bpypoldeo;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/xtrans-1.3.5-ve4eglxuoxe7nsdcyiy4lbcvj2qegxex;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/xextproto-7.3.0-2oagha2n6vf6wmh5lz4o4wxw6zuba6hc;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxcb-1.14-vlmcy2fthy7dzdwwx6n6njw6pmzwbhfg;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/xcb-proto-1.14.1-cimfugd3wmc7sw2wkk2v2fghabo4apg3;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxdmcp-1.1.2-2crqheqqbkcmg4z63zjclasekic5wdls;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxau-1.0.8-2f3vsx23iochykfdq57nbhzaq6zspipj;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/xproto-7.0.31-kyzpwipokb7xsz5f4x4io5tim4p763fp;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/kbproto-1.0.7-xyjeeo2b6j5m2c3xfi5qvv7alxmci3qh;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/inputproto-2.3.2-xbobkehjb4sfrer4oveumgxzln552jgd;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/glproto-1.4.17-sphouu7fembhysc55gv7oovholhmuqcs;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/hsa-rocr-dev-4.5.2-pmkgo6ew56rh2sm7bwaomvsmji6pwhti;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/hsakmt-roct-4.5.2-bnbzssw7brnkeuvwiuapiohawrcaiaqg;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/numactl-2.0.14-e3evcwcbrfcfnv4j2h55y6n6dciu37wi;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libdrm-2.4.100-oubvjpazucmha66spq7cpu277ecpl5mh;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libpthread-stubs-0.4-wngdhpdeuvwyfid6b7rkvbhbm22e4dzr;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/comgr-4.5.2-vform5q7kdm7qlbymgpjo2rwpysytbrx;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/llvm-amdgpu-4.5.2-qb4wlxghtnvdhop7xeyt5657gkbe4gpj;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/z3-4.8.14-bvcp2nq4v7f22lbflvor2m6s7wrsymjp;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/expat-2.4.4-pymrfo6q4hnzthoym5uhxwcvriucuoxh;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libbsd-0.11.3-qkquppz2atadbgdtih5wh3cqo7sdrq6w;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libmd-1.0.3-ahm64asdtstsccavq2szva4guzix3uet;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libelf-0.8.13-o53xuefowaaefem7z2zfar2ftwgyyezy;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/hwloc-2.7.0-hswya4sr4io27hwpjsb7oyhgqb6vzy34;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libxml2-2.9.12-xlbi5rk675mptoyjnuff4q3e2lajs2wr;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/xz-5.2.5-mq4ikt5q4zttamuxqkqe66jlftjbvkf4;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libpciaccess-0.16-4jbn7mznw664dc4mlseouzovsfx6sor5;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/cmake-3.22.2-4oedsu6d3q7kirp2izapy5ygl3cfqsic;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/zlib-1.2.11-ufqpqnpj5gjkfx5sd2o3twwveuowcz2b;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/libiconv-1.16-zth4ohvj7y3p532iqjw7fptg3kkn3ixv;/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/ncurses-6.2-nxjjphalwbf2ywlxpn74nsfcsrq2bbtd' '-DKokkos_ROOT=/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/kokkos-3.5.00-zy73auraurlqnxavzfwyqwiovysfgbdy' '-DARBORX_ENABLE_MPI:BOOL=ON' '/tmp/root/spack-stage/spack-stage-arborx-1.1-rvczpxsyt4gjmti7snqo7dp5jld3wb6v/spack-src'

1 error found in build log:
     4     -- Detecting CXX compiler ABI info
     5     -- Detecting CXX compiler ABI info - done
     6     -- Check for working CXX compiler: /spack/lib/spack/env/gcc/g++ - skipped
     7     -- Detecting CXX compile features
     8     -- Detecting CXX compile features - done
     9     -- Enabled Kokkos devices: SERIAL;HIP
  >> 10    CMake Error at CMakeLists.txt:23 (find_package):
     11      Could not find a package configuration file provided by "rocthrust"
     12      (requested version 2.10.5) with any of the following names:
     13
     14        rocthrustConfig.cmake
     15        rocthrust-config.cmake
     16

See build log for details:
  /tmp/root/spack-stage/spack-stage-arborx-1.1-rvczpxsyt4gjmti7snqo7dp5jld3wb6v/spack-build-out.txt
```
 

### Information on your system

* **Spack:** 0.17.1-1109-d4ca803fc5
* **Python:** 3.8.10
* **Platform:** linux-ubuntu20.04-zen2
* **Concretizer:** clingo

### Additional information

[spack-build-out.txt](https://github.com/spack/spack/files/8042889/spack-build-out.txt)
[spack-build-01-cmake-out.txt](https://github.com/spack/spack/files/8042890/spack-build-01-cmake-out.txt)
[spack-build-env.txt](https://github.com/spack/spack/files/8042898/spack-build-env.txt)

@aprokop @masterleinad 

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/arborx/package.py
+++ b/var/spack/repos/builtin/packages/arborx/package.py
@@ -42,6 +42,7 @@ class Arborx(CMakePackage):
     depends_on('cmake@3.12:', type='build')
     depends_on('cmake@3.16:', type='build', when='@1.0:')
     depends_on('mpi', when='+mpi')
+    depends_on('rocthrust', when='+rocm')
 
     # Standalone Kokkos
     depends_on('kokkos@3.1.00:', when='~trilinos')
@@ -72,6 +73,9 @@ class Arborx(CMakePackage):
             # Only Kokkos allows '+cuda' for now
             options.append(
                 '-DCMAKE_CXX_COMPILER=%s' % spec["kokkos"].kokkos_cxx)
+        if '+rocm' in spec:
+            options.append(
+                '-DCMAKE_CXX_COMPILER=%s' % spec["hip"].hipcc)
 
         return options
 

